app/views/welcome/index.html.erb

```diff
@@ -100,17 +97,24 @@ body {font-family: "Lato", sans-serif}
   <div class="w3-container w3-content w3-center">
     <div class="w3-row w3-padding-32" >
 
+<!-- Visit the Learn More page button -->
+      <div class="w3-panel">
+        <%= link_to "Be The Change. Learn More", "/learn_more", class: "w3-btn w3-light-green w3-block" %>
+      </div>
+
 <!-- Login with Google button -->
       <div class="w3-panel">
         <script src="https://apis.google.com/js/platform.js" async defer></script>
         <%= link_to "Login with Google", "/auth/google_oauth2", class: "w3-btn w3-blue-grey w3-block" %>
       </div>
 
-<!-- Visit the Learn More page button -->
-      <div class="w3-panel">
-        <%= link_to "Be The Change. Learn More", "/learn_more", class: "w3-btn w3-light-green w3-block" %>
+      <div class="w3-panel w3-pale-blue w3-block">
+        <h4 >Why we use Google for Login:</h4>
+        <p class="w3-justify w3-center "> - Google already has strong authorization, helping us protect your data - <br>
```

Comment:
Thank you for adding this. Let's look into making that a partial and adding those CTA's that we spoke with Ian about. 
- Why use Google to login?
- set up a garden
- track your plants
- track your sensor data
- see your carbon impact
- see your Solar Garden community

Reply:
Sounds good.  Let's take all this out and refactor it before merging